caida/caida_test.go

```diff
@@ -44,7 +44,7 @@ var (
 
     // trace for testing
     trace *TraceData
-    pcapFilename = "../resource/10K-test-pkts"
+    pcapFilename = "../resource/equinix-chicago.dirA.20160406-125912.UTC.anon.pcap"
```

Comment:
this line should not be changed. (probably you haven't rebase to the newest master branch)